lib/plugins/inspec-habitat/lib/inspec-habitat/profile.rb

```diff
@@ -1,79 +1,96 @@
 # encoding: utf-8
-# author: Adam Leff
 
 require 'inspec/profile_vendor'
 require 'mixlib/shellout'
 require 'tomlrb'
+require 'ostruct'
 
 module InspecPlugins
   module Habitat
     class Profile
-      attr_reader :options, :path, :profile
-
-      def self.create(path, options = {})
-        creator = new(path, options)
-        hart_file = creator.create
-        creator.copy(hart_file)
-      ensure
-        creator.delete_work_dir
-      end
-
-      def self.setup(path)
-        new(path).setup
-      end
-
-      def self.upload(path, options = {})
-        uploader = new(path, options)
-        uploader.upload
-      ensure
-        uploader.delete_work_dir
-      end
-
       def initialize(path, options = {})
-        @path    = path
+        @path = path
```

Comment:
Very minor thing, but aligning on equal signs makes for easier to read code - here, rubocop has moved it to always be one space on each side. Wah, wah :-)

Reply:
I'm feeling rebellious! I'll align them here 😄